Database/firebase.js

```diff
@@ -0,0 +1,316 @@
+import { db } from "./firebase-config.js";
+import { getDatabase, ref, get, child, onValue, onChildAdded, onChildChanged, onChildRemoved, orderByValue, query, orderByChild } from "firebase/database";
+import { auth } from "./firebase-config.js";
+import { signInAnonymously } from "firebase/auth";
+
+/*
+Authentication
+Signs in user to an anonymous account
+*/
+signInAnonymously(auth)
+  .then(() => {
+    //TO DO
+  })
+  .catch((error) => {
+    const errorCode = error.code;
+    const errorMessage = error.message;
+    console.log(errorCode);
```

Comment:
This should eventually be changed to some page displaying the error code and message and maybe a home/back button to exit the error code page, instead of a console.log()